mysql-test/suite/perfschema/t/bug91541.test

```diff
@@ -0,0 +1,75 @@
+--echo #
+--echo # Test for bug bug91541 ("Flush status" statement adds twice to global values)
+--echo #
+
+--source include/have_perfschema.inc
+
+SET @saved_show_compatibility_56 = @@global.show_compatibility_56;
+SET GLOBAL show_compatibility_56 = OFF;
+
+--echo #
+--echo # default connection
+--echo #
+--let $initial_global_value = query_get_value(SHOW GLOBAL STATUS LIKE 'Slow_queries', Value, 1)
+--let $default_session_value = query_get_value(SHOW SESSION STATUS LIKE 'Slow_queries', Value, 1)
+--let $assert_cond = $default_session_value = 0
```

Comment:
Perhaps it's safe to assume that session Slow_queries are always 0 at the start of the testcase for a bit of simplification and better readability?

Reply:
Agree. Assertion removed.